src/sql/sqlops.proposed.d.ts

```diff
@@ -375,6 +375,16 @@ declare module 'sqlops' {
 				 */
 				content: string | DialogTab[],
 
+				/**
+				 * Register model view content for the dialog
+				 */
+				registerContent(handler: (view: ModelView) => void): void;
+
+				/**
+				 * Returns he model view content
```

Comment:
Typo here. I assume this will return undefined before the dialog has been opened? It might be good to explicitly put that in the comment. Kevin asked me for more thorough API documentation yesterday so I'd suggest we aim to make our comments a bit more descriptive in general